src/assets/translations/zh-hans.yaml

```diff
@@ -1787,7 +1787,7 @@ namespaceList:
 node:
   detail:
     detailTop:
-      containerRuntime: 容器运行时
+      containerEngine: 容器运行时
```

Comment:
This needs to be reverted too, like the English ones. (Same as the other Chinese change.)